### PR TITLE
feat: feature management for logged in users

### PR DESCRIPTION
**Pull request by theogrammer**

### CODE REVIEW

File: auth2.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?php
```
Comment (Tedla2001) [inline auth2.php]: Why is their two versions of auth.php ?

File: auth2.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		fclose($fh);
+		echo "You created your account. Sign in please.";
+		return;
+	}
+
+	// echo '<form method="POST">
+	// 	<input type="email" name="email" placedholder="Email" />
+	// 	<input type="password" name="password" placedholder="password" />
+	// 	<input type="submit">Create Account</button>
+	// </form>';
```
Comment (Tedla2001) [inline auth2.php]: Is this going to be used in the future? If not it should be deleted.

**Review by Tedla2001 — CHANGES_REQUESTED** [review]

File: auth2.php (hunk 1 above)
Comment (theogrammer) [reply auth2.php]: boilerplate code snuck in...will remove, thanks!

File: auth2.php (hunk 2 above)
Comment (theogrammer) [reply auth2.php]: Original auth2 is now removed and this should be gone.

**Review by Tedla2001 — APPROVED** [review]

**Pull request merged**